plugins/modules/user_role_workflow_manager.py

```diff
@@ -1056,14 +1056,13 @@ def valid_user_config_parameters(self, user_config):
         """
         self.log("Validating user configuration parameters...", "INFO")
         error_messages = []
+        regex_name_validation = re.compile(r"^[A-Za-z0-9_-]+$")
```

Comment:
we can also set message and pass it. So it will be easy to maintain...

    regex_name_validation_msg = "must only contain letters, numbers, underscores, and hyphens and should not contain spaces or other special characters."